docs/0.3/purchase_order_family_specification.md

```diff
@@ -37,42 +35,57 @@ Order.
 The state attributes of a purchase order include:
 
 * Unique identifier
-* Workflow Status
-* Indicator if the purchase order has been closed
-* Unique identifier of an accepted version of this purchase order
+* Workflow state
+* Buyer organization identifier
+* Seller organization identifier
 * List of all versions of this purchase order
+* Unique identifier of an accepted version of this purchase order
+* Alternate identifiers for this purchase order
 * Timestamp from when the purchase order was created
+* Indicator if the purchase order has been closed
+* Workflow name
 
-The protocol buffer is defined as follows:
+The `PurchaseOrder` protocol buffer is defined as follows:
 
 ```protobuf
   message PurchaseOrder {
       // The unique identifier of the purchase order
-      required string uuid = 1;
-      // The workflow status of the purchase order
-      required string workflow_status = 2;
+      string uuid = 1;
+      // The workflow state of the purchase order
+      string workflow_state = 2;
+      // Organization identifier of buyer
+      string buyer_org_id = 3;
+      // Organization identifier of seller
+      string seller_org_id = 4;
       // List of all versions of the purchase order
-      repeated PurchaseOrderVersion versions = 3;
+      repeated PurchaseOrderVersion versions = 5;
       // Unique identifier of an accepted purchase order version
-      string accepted_version_number = 4;
+      string accepted_version_number = 6;
+      // Alternate identifiers for this purchase order
+      repeated PurchaseOrderAlternateId alternate_ids = 7;
       // Time the purchase order was created.
-      uint64 created_at = 5;
+      uint64 created_at = 8;
       // True if the purchase order was closed, false otherwise
-      bool is_closed = 6;
+      bool is_closed = 9;
+      // The name of the workflow of this purchase order
+      string workflow_type = 10;
```

Comment:
Please update this spec to align with the most current version of the RFC

Reply:
Done